libs/cohere/tests/integration_tests/test_rerank.py

```diff
@@ -22,3 +23,17 @@ def test_langchain_cohere_rerank_documents() -> None:
     test_query = "Test query"
     results = rerank.rerank(test_documents, test_query)
     assert len(results) == 2
+
+
+@pytest.mark.vcr()
+def test_langchain_cohere_rerank_with_rank_fields() -> None:
+    rerank = CohereRerank()
+    test_documents = [
+        {"content": "This document is about Penguins.", "subject": "Physics"},
+        {"content": "This document is about Physics.", "subject": "Penguins"},
+    ]
+    test_query = "penguins"
+    response = rerank.rerank(test_documents, test_query, rank_fields=["content"])
+    results = {r["index"]: r["relevance_score"] for r in response}
+    assert len(results) == 2
+    assert results[0] > results[1]
```

Comment:
maybe assert on the top document in the response matching the "penguin body, physics subject" one?

Reply:
great idea - will add it